Adyen/UI/View Controllers/AddressLookup/Form/AddressInputFormViewController+ViewModel.swift

```diff
@@ -36,15 +41,17 @@ extension AddressInputFormViewController {
         ///   - completionHandler: A closure that takes an optional address.
         ///    It's the presenters responsibility to dismiss the viewController.
         public init(
+            for addressType: FormAddressPickerItem.AddressType,
             style: FormComponentStyle,
             localizationParameters: LocalizationParameters?,
             initialCountry: String,
             prefillAddress: PostalAddress?,
             supportedCountryCodes: [String]?,
             addressViewModelBuilder: AddressViewModelBuilder = DefaultAddressViewModelBuilder(),
-            handleShowSearch: ShowSearchHandler?,
+            handleShowSearch: ShowSearchHandler? = nil,
```

Comment:
I guess it's too late to change it since it may be a breaking change, but I believe it's less confusing to use `showSearchHandler` than `handleShowSearch`. Just my perception when I read it for the first time.

Reply:
Not too late but felt like the right property name because the `ShowSearchHandler` is a typealias for a closure `public typealias ShowSearchHandler = (_ currentInput: PostalAddress) -> Void` - any thoughts?